examples/plot_intersectional_bias.py

```diff
@@ -0,0 +1,1327 @@
+#
+# In this case, **false negative predictions** would probably be most relevant, because when
+# classifying tumours as being cancerous or non-cancerous, we would prefer to mistake a benign
+# tumour as being cancerous, as compared to the other way around. This is called a false positive
+# (or misdiagnosis), and it’s better than a false negative (or underdiagnosis) in this specific
+# circumstance.
+#
+# Why? Because a false negative would result in a false perception that the patient does not have
+# cancer, which could delay potentially life-saving treatment.
+#
+# There is a study suggesting that, when diagnosing the nature of tumours from medical images,
+# false negative rates are higher for disadvantaged or underserved groups defined by intersecting
+# features (e.g., low-income, Black women). :footcite:`seyyedkalantari2021underdiagnosis` This is an excellent,
+# real-world example of how the application of ML in healthcare can *amplify harms*, if deployed
+# without concern for fairness.
+#
+# *Which performance metric is most important in our case?*
+#
+# In our case, the opposite would be true - a false positive or misdiagnosis is more harmful for
+# our hypothetical scenario, because it would lead to unnecessary treatment for schizophrenia being
+# administered, as well as other negative impacts of a schizophrenia diagnosis, such as stigma or a
+# poorer perceived prognosis. While we want every patient to be appropriately diagnosed, clinicians
+# generally agree that it is better to be misdiagnosed with an affective disorder as compared to
+# schizophrenia. Indeed, when diagnosing patients, clinicians tend to rule out any factors that
+# could lead to psychosis or symptoms of schizophrenia, such as trauma, substance use, and
+# affective disorder, which in some cases, can have psychotic features.
+#
+# So in our example, we’d like to evaluate whether rates of misdiagnosis or **false positives** are
+# the same across patient groups. False positive rates are calculated by dividing false positive
+# predictions by all negative predictions (:math:`FPR = \frac{FP}{FP + TN}`)
+#
+# Fairness metrics
+# ~~~~~~~~~~~~~~~~
+#
+# When evaluating ML models for fairness, we typically examine the ratio of a given performance
+# metric between two groups of interest, and whether it is greater or less than true parity (1).
+# The numerator in this equation is the metric for the group we are interested in evaluating (on
+# top), and the denominator is the same metric for the reference group (bottom).
+#
+#   :math:`\text{Relative Metric} = \frac{\text{Metric }_{\text{group of interest}}}{\text{Metric
+#   }_{\text{reference group}}}`
+#
+# If this ratio is greater than 1, then the metric is higher in the group of interest vs reference.
+#
+# If this ratio is less than 1, then the metric is lower in the group of interest vs reference.
+#
+# Fairness metrics themselves can be broadly divided into 3 categories:
```

Comment:
Can you provide a citation for this statement?

Reply:
Taken from here: https://fairmlbook.org/classification.html#statistical-non-discrimination-criteria

Reply:
added the reference